loopr-ios/Setting/SettingManageWallet/SettingWalletDetail/SettingWalletDetailViewController.swift

```diff
@@ -56,13 +56,15 @@ class SettingWalletDetailViewController: UIViewController, UITableViewDelegate,
 
             // Pop to SettingViewController
```

Comment:
In the current implementation, app will always jump to WalletViewController after the current wallet is changed. During the switch, a needRelaunchCurrentAppWallet will be published. WalletViewController will receive the notification and send a getBalanceAndPriceQuoteAndNonce API call.

Reply:
getBalanceAndPriceQuoteAndNonce will get the latest nonce using the address in AppWallet. So it won't get the nonce with the incorrect address.